File: Task-4/SwapNumbers.java
public class SwapNumbers{
  public static void main(String[] args){
    int firstNumber = 5, secondNumber = 7;
    System.out.println("Before Swap:");
    System.out.println("First Number = " + firstNumber + " Second Number = "+ secondNumber);
    swapNumbers(firstNumber, secondNumber);
  }
  
  static void swapNumbers(int firstNumber, int secondNumber){
    firstNumber = firstNumber - secondNumber;
    secondNumber = firstNumber + secondNumber;
    firstNumber = secondNumber - firstNumber;
    System.out.println("After Swap:");
    System.out.println("First Number = " + firstNumber + " Second Number = "+ secondNumber);
  }
}



/*prints
Before Swap:
First Number = 5 Second Number = 7
After Swap:
First Number = 7 Second Number = 5
*/

File: Task-4/SwapNumbers.py
def swapNumbers(firstNumber,secondNumber):
  firstNumber = firstNumber + secondNumber 
  secondNumber = firstNumber - secondNumber
  firstNumber = firstNumber - secondNumber
    
  print("After Swap:")
  print("Value of first number:", firstNumber)
  print("Value of second number:", secondNumber)
  
  
firstNumber = 10
secondNumber = 20

print("Before Swap:")
print("Value of first number:",firstNumber)
print("Value of second number:",secondNumber)
swapNumbers(firstNumber,secondNumber)
